os_migrate/roles/conversion_host/tasks/conv_host_inventory.yml

```diff
@@ -24,6 +24,7 @@
     name: "{{ inventory_name }}"
     groups:
       - conversion_hosts
+      - "conversion_hosts_{{ inventory_name.split('_')[-1] }}"
```

Comment:
Can we add another key with the group name below the inventory_name specification where this is called (in the other file)? So that we have the group explicit, rather than relying on name magic here and a hidden dependency on how the conversion hosts must be named.